src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -77,7 +78,8 @@ public PersonCard(Person person, int displayedIndex) {
         if (person.getSocialHandle().value.isEmpty()) {
             socialHandle.setManaged(false);
         }
-        gender.setText(person.getGender().gender);
+        gender.setText("");
+        gender.setGraphic(new ImageView(person.getGender().genderIcon));
```

Comment:
I am wondering if it would be better to have a seperate class in UI that takes the in the gender as an argument and returns the genderIcon as I feel that Icon is not exactly a property of Gender.